views/partials/compromised_accounts.hbs

```diff
@@ -14,5 +14,5 @@
     </div>
   {{/each}}
 
-  <span class="source-info">Breach data provided by <span class="demi">Have I Been Pwned?</span></span>
+  <span class="source-info">Breach data provided by <a class="demi compromised" href="https://www.haveibeenpwned.com" alt="Have I Been Pwned">Have I Been Pwned</a></span>
```

Comment:
For reasons currently unknown, i can't test this path. When I search a compromised email on localhost, it isn't returning any results (even if I run the <kbd>$ node scripts/add-breached-emails.js</kbd> script).